sui-execution/latest/sui-verifier/src/private_generics.rs

```diff
@@ -25,18 +25,14 @@ pub const PUBLIC_TRANSFER_FUNCTIONS: &[&IdentStr] = &[
     ident_str!("public_share_object"),
     ident_str!("public_receive"),
     ident_str!("receiving_object_id"),
+    ident_str!("public_multiparty_transfer"),
```

Comment:
It may _may_ be useful to put a comment on this constant mentioning that the order of at least the first four elements of the array is important (or at least must stay the first four elements).